src/portal/views/Dao.vue

```diff
@@ -34,90 +29,88 @@
 
       <v-card
         class="my-3 pa-3"
-        v-for="(proposal,i) in filteredProposals()"
+        v-for="(proposal, i) in filteredProposals()"
         :key="i"
       >
         <v-card-title>
-          {{proposal.action.toUpperCase()}}
+          {{ proposal.action.toUpperCase() }}
         </v-card-title>
         <v-card-subtitle>
-          {{proposal.hash}}
+          {{ proposal.hash }}
         </v-card-subtitle>
         <v-card-text>
-          {{proposal.description}}
-          <a
-            v-bind:href="proposal.link"
-            v-bind:target="'blank'"
-          >More details</a>
-
+          {{ proposal.description }}
+          <a v-bind:href="proposal.link" v-bind:target="'blank'"
+            >More details</a
+          >
         </v-card-text>
 
-        <v-container
-          fluid
-          class="d-flex justify-space-between my-5"
-        >
+        <v-container fluid class="d-flex justify-space-between my-5">
           <v-btn
-            color="green"
-            class=" "
-            :width="`${proposal.ayes.length *100 + 100}`"
+            color="primary"
+            class=""
+            width="100px"
             @click="openVoteDialog(proposal.hash, true)"
             :loading="loadingVote"
-          >Yes
-            <v-divider
-              class="mx-3"
-              vertical
-            />{{proposal.ayes.length}}
+            >Yes <v-divider class="mx-3" vertical />{{ proposal.ayes.length }}
           </v-btn>
 
-          <div class="d-flex align-center  text-center threshold">
+          <div class="d-flex align-center text-center threshold">
             <v-divider vertical />
-            <span>Threshold: <br>{{(proposal.nayes.length + proposal.ayes.length)}}/{{proposal.threshold}}
-
+            <span
+              >Threshold: <br />{{
+                proposal.nayes.length + proposal.ayes.length
+              }}/{{ proposal.threshold }}
             </span>
             <v-divider vertical />
-
           </div>
           <v-btn
-            color="red"
-            :width="`${proposal.nayes.length *100 + 100}`"
+            color="grey lighten-2 black--text"
+            width="100px"
```

Comment:
hey, @samaradel please do not remove the width related to the number of ayes and nays on a proposal as it should display as the proposal's nays and ayes are trying to reach the threshold.